### DISCUSSION TITLE
Nice work

### DISCUSSION THREAD

**Discussion by Mahoney0101** [bavix/gripmock#359] [General]
Hi, I'm currently looking at using this project to simulate some services.
The addition of the UI and yaml format for stubs are a nice touch. 
I'm just curious as to what happened with original repository or why these additions weren't added to the original?
I see the original repo is fairly inactive

**Comment by rez1dent3** [comment]
Hello. Thank you very much for your feedback, I really appreciate it.

The original is developing, but slowly. I forked it just because of one bug: https://github.com/tokopedia/gripmock/pull/148
I waited for the fix to be implemented, but it didn't happen. Now I continue development on my own.

**Reply by rez1dent3** [reply]
> The addition of the UI 

The UI will be in 3.x, but it is still raw. There is no adding, only browsing. I still have a lot of work to do)

For now you can wish me luck :D

**Reply by Mahoney0101** [reply]
Good Luck!
